File: backend/Models/Student.cs
namespace SkillwaveAPI.Models
{
    public class Student
    {
        public string? Id { get; set; }
        public string? FirstName { get; set; }
        public string? LastName { get; set; }
        public string? Email { get; set; }
        public string? Phone { get; set; }
        public string? Password { get; set; }
        public DateTime CreatedAt { get; set; }
        public string? PhotoPath { get; set; } // قد يكون اختيارياً
        public string Roll { get; set; }

    }
}

File: learning-platform/src/app/models/student.ts
// export interface Student {
//   id: string;           // معرف الطالب
//   firstName: string;    // الاسم الأول
//   lastName: string;     // الاسم الأخير
//   email: string;        // البريد الإلكتروني
//   password: string;     // كلمة المرور
//   phone?: string;       // رقم الهاتف (اختياري
//   PhotoBase64?: string;
//   createdAt?: Date;     // تاريخ الإنشاء
// }
export interface Student {
  id: string;
  firstName: string;
  lastName: string;
  email: string;
  phone?: string;
  password: string;
  createdAt: string;
  profileImage?: string; // يمكن أن يكون اختياريًا
}
